04-spatial-operations.Rmd

```diff
@@ -17,28 +17,27 @@ Spatial operations are a vital part of geocomputation.
 This chapter shows how spatial objects can be modified in a multitude of ways based on their location and shape.
 The content builds on the previous chapter because many spatial operations have a non-spatial (attribute) equivalent.
 This is especially true for *vector* operations: Section \@ref(vector-attribute-manipulation) on vector attribute manipulation provides the basis for understanding its spatial counterpart, namely spatial subsetting (covered in Section \@ref(spatial-subsetting)).
-Spatial joining (Section \@ref(spatial-joining)) and aggregation (\@ref(spatial-aggr)) also have non-spatial counterparts, covered in the previous chapter.
+Spatial joining (Section \@ref(spatial-joining)) and aggregation (Section \@ref(spatial-aggr)) also have non-spatial counterparts, covered in the previous chapter.
 
 Spatial operations differ from non-spatial operations in some ways, however.
 To illustrate the point, imagine you are researching road safety.
 Spatial joins can be used to find road speed limits related with administrative zones, even when no zone ID is provided.
-<!-- This could be a good example that - a candidate to replace the rather contrived asia example below. -->
 But this raises the question: should the road completely fall inside a zone for its values to be joined?
 Or is simply crossing or being within a certain distance sufficient?
-When posing such questions it becomes apparent that spatial operations differ substantially from attribute operations on data frames:
+When posing such questions, it becomes apparent that spatial operations differ substantially from attribute operations on data frames:
 the *type* of spatial relationship between objects must be considered.
 These are covered in Section \@ref(topological-relations), on topological relations.
 
 Another unique aspect of spatial objects is distance.
 All spatial objects are related through space and distance calculations, covered in Section \@ref(distance-relations), can be used to explore the strength of this relationship.
 
-Naturally, we can also subset *rasters* based on location and coordinates (Section \@ref(spatial-raster-subsetting)) and merge different raster tiles into one raster (covered in Section \@ref(merging-rasters)). 
-The most important spatial operation on raster data, however, is *map algebra*. 
-Map algebra makes raster processing very elegant and fast (covered in sections \@ref(map-algebra) to \@ref(global-operations-and-distances)).
-Map algebra is also the prerequisite for distance calculations on rasters \@ref(global-operations-and-distances).
+Spatial operations also apply to raster objects.
+Spatial subsetting of raster objects is covered in Section \@ref(spatial-raster-subsetting); merging several raster 'tiles' into a single, more geographically extensive object is covered in Section \@ref(merging-rasters).
```

Comment:
I think not. If the 'more geographically extensive' clause were removed it would read:

> merging several raster 'tiles' into a single object, is covered

so if anything the comma should be before 'object', e.g.:

> merging several raster 'tiles' into a single, more geographically extensive, object is covered in Section

but that also does not work.

Will change to this, which is shorter and simpler yet looses little meaning:

```
Spatial subsetting of raster objects is covered in Section \@ref(spatial-raster-subsetting); merging several raster 'tiles' into a single object is covered in Section \@ref(merging-rasters).
```